src/tools/cephfs_mirror/FSMirror.cc

```diff
@@ -431,6 +431,9 @@ void FSMirror::add_peer(const Peer &peer) {
   }
   m_peer_replayers.emplace(peer, std::move(replayer));
   ceph_assert(m_peer_replayers.size() == 1); // support only a single peer
+  m_service_daemon->add_or_update_peer_attribute(m_filesystem.fscid, peer,
```

Comment:
its likely that I'm missing something, but how passing in `false` boolean help here? Don't we have to remove the (failure) itself?